app/views/responsible_body/internet/mobile/extra_data_requests/show.html.erb

```diff
@@ -0,0 +1,27 @@
+<%- content_for :before_content do %>
+  <%= govuk_link_to('Back', responsible_body_internet_mobile_extra_data_requests_path, class: 'govuk-back-link') %>
+<% end %>
+<%- title = @request.account_holder_name %>
```

Comment:
We need to be careful with this – this title can end up in Google Analytics, which would be PII.

Any suggestions for an alternative title?
- Just first name? (split on first space)
- First 5 chars of phone number?

Reply:
Hmm, tricky!

I think if that's a concern (GA includes IP addresses which are also classed as PII, for instance) then we need to have different values for title tag vs the H1 tag on the page. The H1 as designed seems like the right choice, but if we need a non-PII title tag then we need something that uniquely identifies that page (== that request)  but not the person it represents.

First 5 chars of phone number would likely be shared among many requests. 
Likewise first name - and we if we were splitting at a space, we could easily accidentally expose the full name if they've mis-typed, or written it in native non-latin characters like Chinese.

Maybe something like "request #(id) - (_last_ 5 chars of phone number)" ?

Reply:
Our GA should be using the `anonymizeIp` setting: https://developers.google.com/analytics/devguides/collection/gtagjs/ip-anonymization

Full names are against GA's terms in https://support.google.com/analytics/answer/7686480?hl=en

`[network name] request - (07 … last 4 chars of phone number)`
example: `O2 request – (07…2342)`

We need the `07` so it's clear it's a number

Reply:
Yep, I like that - done in 9033173d :+1: 
I've added a separate [Trello card](https://trello.com/c/5GWA2q1W/1505-anonymise-ips-in-google-analytics) to make sure the IPs are anonymised